.ci/update-github-page.sh

```diff
@@ -93,4 +93,5 @@ curl \
   -X PATCH https://api.github.com/repos/checkstyle/checkstyle/releases/"$RELEASE_ID" \
   -H "Accept: application/vnd.github+json" \
   -H "Authorization: token $BUILDER_GITHUB_TOKEN" \
-  -d "${JSON}"
+  -d "${JSON}" \
+  --fail-with-body
```

Comment:
Please move to line with `curl`, typically data argument is last, preceded by headers, preceded by url and request method.

Reply:
Done.